src/commands/auth/delete.ts

```diff
@@ -0,0 +1,74 @@
+/*********************************************************************
+ * Copyright (c) 2020 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+import { Command, flags } from '@oclif/command'
+import { cli } from 'cli-ux'
+
+import { CheApiClient } from '../../api/che-api-client'
+import { CheServerLoginManager } from '../../api/che-login-manager'
+import { CHE_API_ENDPOINT_KEY, username, USERNAME_KEY } from '../../common-flags'
+
+export default class Delete extends Command {
+  static description = 'Delete specified login session(s)'
+
+  static args = [
+    {
+      name: CHE_API_ENDPOINT_KEY,
+      description: 'Eclipse Che server API endpoint',
+      env: 'CHE_API_ENDPOINT',
+      required: true
+    }
+  ]
+  static flags: flags.Input<any> = {
+    help: flags.help({ char: 'h' }),
+    [USERNAME_KEY]: username,
+  }
+
+  static examples = [
+    '# Delete given user login information for specified cluster:\n' +
+    'auth:delete che-che.apps-crc.testing/api -u username',
+    '\n# Delete all existing logins for specified cluster:\n' +
+    'auth:delete che-che.apps-crc.testing',
+  ]
+
+  async run() {
+    const { args, flags } = this.parse(Delete)
+
+    let cheApiEndpoint = CheApiClient.normalizeCheApiEndpointUrl(args[CHE_API_ENDPOINT_KEY])
+    let username: string | undefined = flags[USERNAME_KEY]
+
+    const loginManager = await CheServerLoginManager.getInstance(this.config.configDir)
+
+    if (!loginManager.hasLoginFor(cheApiEndpoint)) {
+      // Maybe /api suffix isn't provided
+      const cheApiEndpointGuess = cheApiEndpoint + '/api'
+      if (!loginManager.hasLoginFor(cheApiEndpointGuess)) {
+        cli.info(`No registered logins on server ${cheApiEndpoint}`)
+        return
+      }
+      cheApiEndpoint = cheApiEndpointGuess
+    }
+
+    if (username) {
+      if (!loginManager.hasLoginFor(cheApiEndpoint, username)) {
+        cli.info(`No existing logins for ${username} on server ${cheApiEndpoint}`)
+        return
+      }
+    }
+
+    loginManager.deleteLoginContext(cheApiEndpoint, username)
+    if (username) {
+      cli.info(`Succesfully logged out ${username} on ${cheApiEndpoint}`)
+    } else {
+      cli.info(`Succesfully logged out of ${cheApiEndpoint}`)
```

Comment:
`of` -> `on`

Reply:
Login sessions on ${cheApiEndpoint} are deleted.

Reply:
I thuink these two messages should be consistent. But I agree, we need to stress that all sessions were deleted.